cypress/e2e/interface_builder.cy.js

```diff
@@ -13,8 +13,14 @@ const parseMappingOptions = (mapping) => {
 };
 
 const setupInterfaceEditorFromSource = (iface) => {
-  cy.get('#interfaceSource').clear().paste(JSON.stringify(iface));
-  cy.wait(500);
+  cy.get('.monaco-editor')
+    .should('be.visible')
+    .then(() => {
+      cy.window().then((win) => {
+        const editor = win.monaco.editor.getModels()[0];
+        editor.setValue(JSON.stringify(iface, null, 4));
+      });
+    });
```

Comment:
It would be ideal to stick to the basic Cypress commands, thus simulating an actual user interacting and writing (or pasting) into the editor.
The main idea when doing e2e tests is to avoid thinking about the implementation and just test the expected user interaction.
Do you think it's feasible, or did you encounter some issues?